packages/grid/x-grid-modules/src/models/api/columnApi.ts

```diff
@@ -51,6 +51,7 @@ export interface ColumnApi {
    * Allows to batch update multiple columns at the same time.
    *
    * @param cols [[ColDef[]]]
+   * @param resetState
    */
-  updateColumns: (cols: ColDef[]) => void;
+  updateColumns: (cols: ColDef[], resetState?: boolean) => void;
```

Comment:
https://github.com/mui-org/material-ui-x/issues/190

```suggestion
  updateColumns: (cols: ColDef[]) => void;
```

Reply:
Unfortunately do need that flag in the method definition itself in order to force update the columns definition array. As far as I can see if I remove it from here this will not be available for clients, yes? So basically we will not expose that option?